go/vt/wrangler/vexec_test.go

```diff
@@ -176,7 +176,7 @@ func TestWorkflowListStreams(t *testing.T) {
 	logger := logutil.NewMemoryLogger()
 	wr := New(logger, env.topoServ, env.tmc)
 
-	_, err := wr.ShowWorkflow(ctx, workflow, keyspace)
```

Comment:
It will be better to continue to call this but assert on the returned result

Reply:
The point of this test though is to check the logger output, which will only exist if we call one level up.